app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/views/helpers/GlobalSection.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apigatekeeperapprovalsfrontend.views.helpers
+
+object GlobalSection {
+
+  import uk.gov.hmrc.apigatekeeperapprovalsfrontend.views.helpers.FormKeys._
+
+  private def a(em: String)(found: String => String, notFound: () => String) = {
```

Comment:
is there a good  descriptive name we could give this?

Reply:
I'm gonna leave this as it is so that we keep consistency with TPDFE, which is where this came from :)